### PR TITLE
Adds client static values

### PR DESCRIPTION
**Pull request by NejcZdovc**
Resolves https://github.com/brave/brave-browser/issues/6304

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:
- no test plan needed

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: vendor/brave-ios/Ledger/BATBraveLedger.mm [hunk 1]
```diff
@@ -965,6 +965,37 @@ - (void)clearState:(const std::string&)name
   [self savePrefs];
 }
 
+- (bool)getBooleanOption:(const std::string&)name
+{
+  // TODO implement
```
Comment (kylehickinson) [inline vendor/brave-ios/Ledger/BATBraveLedger.mm]: Is there going to be globally defined strings for checking? Future PR?

Comment (NejcZdovc) [reply vendor/brave-ios/Ledger/BATBraveLedger.mm]: yes in new PR I will add examples of usage, but every platform can implement their own version of it

Comment (NejcZdovc) [reply vendor/brave-ios/Ledger/BATBraveLedger.mm]: on desktop we will have `map<string, type>` and simple `at` to get it

Comment (NejcZdovc) [reply vendor/brave-ios/Ledger/BATBraveLedger.mm]: https://github.com/brave/brave-core/pull/3628 client implementation

**Pull request merged**